### PR TITLE
Update dlp example code path in README

### PR DESCRIPTION
**Pull request by matthewmarr**
Update dlp example code path in README

### CODE REVIEW

**Comment by matthewmarr** [comment]
@ocervell Please review

File: examples/dlp/cloud_function_example/cloud_function/main.py [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+
+from google.cloud import dlp_v2
+from google.cloud import pubsub_v1
+
+_INFO_TYPE_LIST = ['CREDIT_CARD_NUMBER', 'DATE_OF_BIRTH', 'EMAIL_ADDRESS']
+_OUTPUT_TOPIC_NAME = os.environ['OUTPUT_TOPIC_NAME']
+_PROJECT_ID = os.environ['GCP_PROJECT']
+
+
+def process_log_entry(event, unused_context):
```
Comment (jaketf) [inline examples/dlp/cloud_function_example/cloud_function/main.py]: Current style recommendations are that you delete this unused arg in the first line of the function. See [go/python-style](http://go/pystyle#214-decision).

Add:
```
del unused_context # The cloud functions context is unused.
```

File: examples/dlp/cloud_function_example/cloud_function/main.py [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+
+    Args:
+      event: A Pub/Sub event.
+
+    Returns:
+      The json_payload from a pub/sub message.
+  """
+
+  pubsub_message = base64.b64decode(event['data']).decode('utf-8')
+  json_payload = json.loads(pubsub_message)
```
Comment (jaketf) [inline examples/dlp/cloud_function_example/cloud_function/main.py]: Shorten this to `return json.loads(pubsub_message)`

File: examples/dlp/cloud_function_example/cloud_function/main.py [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+  return json_payload
+
+
+def _build_deidentify_config(info_type_list):
```
Comment (jaketf) [inline examples/dlp/cloud_function_example/cloud_function/main.py]: Add unit test for this function.

File: examples/dlp/cloud_function_example/cloud_function/main.py [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+def _post_scrubbed_message_callback(message_future):
+  """The pub/sub message post callback.
+
+    Args:
+      message_future: A google.api_core.future.Future object returned from the
+        pub/sub publish call.
+  """
+
+  if message_future.exception(timeout=30):
+    print('Publishing message on {} threw an Exception {}.'.format(
```
Comment (jaketf) [inline examples/dlp/cloud_function_example/cloud_function/main.py]: I know that CFs will log stdout but refactor to use `logging.info` for clarity.

File: examples/dlp/cloud_function_example/cloud_function/main.py [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+        _OUTPUT_TOPIC_NAME, message_future.exception()))
+  else:
+    print("Posted message id:" + message_future.result())
```
Comment (jaketf) [inline examples/dlp/cloud_function_example/cloud_function/main.py]: Refactor to be `logging.info`

File: examples/dlp/cloud_function_example/cloud_function/requirements.txt [hunk 6]
```diff
@@ -0,0 +1,2 @@
+google-cloud-dlp
```
Comment (jaketf) [inline examples/dlp/cloud_function_example/cloud_function/requirements.txt]: Add minimum versions with `>=`

File: examples/dlp/cloud_function_example/README.md [hunk 7]
```diff
@@ -0,0 +1,130 @@
...
+* Write scrubbed logs to a Pub/Sub topic.
+
+### Prerequisites
+
+Ensure that you have gcloud installed and authenticated to the project you want
+to deploy the example to.
+
+### Enable Required APIs
+
+The Cloud Functions, Pub/Sub, and DLP APIs will all need to be enabled for this
```
[the remark this answers is not in the document]
Comment (matthewmarr) [reply examples/dlp/cloud_function_example/README.md]: I included the gcloud command, will that suffice? I wasn't able to successfully get the url to work in testing.

File: examples/dlp/cloud_function_example/README.md [hunk 8]
```diff
@@ -0,0 +1,130 @@
...
+These instructions will walk you through setting up your environment to do the
+following:
+
+* Export logs to a Pub/Sub log export
+* Deploy a Cloud Function that subscribes to the log export
+* Write scrubbed logs to a Pub/Sub topic.
+
+### Prerequisites
+
+Ensure that you have gcloud installed and authenticated to the project you want
```
[the remark this answers is not in the document]
Comment (matthewmarr) [reply examples/dlp/cloud_function_example/README.md]: Done

File: examples/dlp/cloud_function_example/README.md [hunk 9]
```diff
@@ -0,0 +1,130 @@
...
+Download the required files.
+```
+mkdir dlp_cloud_function_example
+
+wget -O dlp_cloud_function_example/main.py https://github.com/GoogleCloudPlatform/professional-services/blob/master/examples/dlp/cloud_function_example/requirements.txt
+
+wget -O dlp_cloud_function_example/requirements.txt https://github.com/GoogleCloudPlatform/professional-services/blob/master/examples/dlp/cloud_function_example/main.py
+```
+
+Deploy the Cloud Function
```
[the remark this answers is not in the document]
Comment (matthewmarr) [reply examples/dlp/cloud_function_example/README.md]: Done

File: examples/dlp/cloud_function_example/README.md [hunk 10]
```diff
@@ -0,0 +1,130 @@
...
+gcloud projects add-iam-policy-binding $PROJECT_ID \
+  --member $SERVICE_ACCOUNT \
+  --role roles/pubsub.publisher
+```
+
+### Cloud Function
+
+Cloud Functions will be used to call the DLP API to scrub the log content, then post the output to a new Pub/Sub topic.
+
+Download the required files.
```
[the remark this answers is not in the document]
Comment (matthewmarr) [reply examples/dlp/cloud_function_example/README.md]: Done

File: examples/dlp/cloud_function_example/README.md [hunk 11]
```diff
@@ -0,0 +1,130 @@
...
+  --runtime python37 \
+  --trigger-topic $LOG_EXPORT_TOPIC_NAME \
+  --entry-point process_log_entry \
+  --set-env-vars OUTPUT_TOPIC_NAME=$DLP_SCRUBBED_TOPIC_NAME \
+  --source dlp_cloud_function_example/.
+```
+
+Wait a few minutes for the Cloud Function to deploy, then you can proceed to test.
+
+### Testing the DLP API
```
[the remark this answers is not in the document]
Comment (matthewmarr) [reply examples/dlp/cloud_function_example/README.md]: Done

File: examples/dlp/cloud_function_example/cloud_function/main.py (hunk 2 above)
Comment (matthewmarr) [reply examples/dlp/cloud_function_example/cloud_function/main.py]: Done

File: examples/dlp/cloud_function_example/cloud_function/main.py (hunk 3 above)
Comment (matthewmarr) [reply examples/dlp/cloud_function_example/cloud_function/main.py]: Done

File: examples/dlp/cloud_function_example/cloud_function/main.py (hunk 4 above)
Comment (matthewmarr) [reply examples/dlp/cloud_function_example/cloud_function/main.py]: Done

File: examples/dlp/cloud_function_example/cloud_function/main.py (hunk 5 above)
Comment (matthewmarr) [reply examples/dlp/cloud_function_example/cloud_function/main.py]: Done

File: examples/dlp/cloud_function_example/cloud_function/requirements.txt (hunk 6 above)
Comment (matthewmarr) [reply examples/dlp/cloud_function_example/cloud_function/requirements.txt]: Done

File: examples/dlp/cloud_function_example/README.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (jaketf) [reply examples/dlp/cloud_function_example/README.md]: yeah that's fine.

**Comment by jaketf** [comment]
@matthewmarr LGTM.
